core/controllers/pages_test.py

```diff
@@ -15,15 +15,148 @@
 """Tests for various static pages (like the About page)."""
 
 from core.tests import test_utils
+import feconf
 
 
-class NoninteractivePagesTests(test_utils.GenericTestBase):
+class AboutPageTest(test_utils.GenericTestBase):
 
     def test_about_page(self):
-        """Test the About page."""
+        """Test for About page."""
         response = self.testapp.get('/about')
         self.assertEqual(response.status_int, 200)
         self.assertEqual(response.content_type, 'text/html')
         response.mustcontain(
             'I18N_ABOUT_PAGE_CREDITS_TAB_HEADING',
             'I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_5_LICENSE_HEADING')
+
+
+class SplashPageTest(test_utils.GenericTestBase):
+
+    def test_splash_page(self):
+        """Test for splash page."""
+        response = self.testapp.get('/splash')
+        self.assertEqual(response.status_int, 200)
+        self.assertEqual(response.content_type, 'text/html')
+
+    def test_splash_page_with_c_value(self):
+        """Test for splash page when c value is included."""
+        response = self.testapp.get('/splash?c=c')
+        self.assertEqual(response.status_int, 302)
+        self.assertEqual(response.content_type, 'text/html')
+
```

Comment:
Can you add something similar to line 28 to check that splash page is loading correctly? (Ditto everywhere!)